src/script/notification/NotificationRepository.ts

```diff
@@ -259,7 +259,9 @@ export class NotificationRepository {
             const messageInfo = messageId
```

Comment:
all the edits in this file are just improving some logger [object Object] issues